File: doc/user/06_feature/scripting/rosetta/13_hello_dad/ex.pl
sub hello
{
	dad('heldi', 'new');
	dad('heldi', 'begin_vbox');
		dad('heldi', 'label', 'Hello');
		dad('heldi', 'label', 'world!');
	dad('heldi', 'end');
	dad('heldi', 'run_modal', 'hello-world', '');
}

fgw_func_reg('hello');

File: doc/user/06_feature/scripting/rosetta/12_hello_dad/ex.pas
function hello();
begin
	dad('heldi', 'new');
	dad('heldi', 'label', 'Hello world!');
	dad('heldi', 'button_closes', 'close', 0);
	dad('heldi', 'run_modal', 'hello-world', 'Hello world dialog');
end;

function main(ARG);
begin
	fgw_func_reg(hello);
end;
